compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/TilingInterfaceImpl.cpp

```diff
@@ -1704,45 +1704,51 @@ AttentionOp::getTiledImplementation(OpBuilder &builder,
 
   Location loc = getLoc();
 
-  SmallVector<Range> querySlice =
-      getPermutedSlice(getQueryMap(), offsets, sizes);
-  SmallVector<Range> keySlice = getPermutedSlice(getKeyMap(), offsets, sizes);
-  SmallVector<Range> valueSlice =
-      getPermutedSlice(getValueMap(), offsets, sizes);
-  SmallVector<Range> outputSlice =
-      getPermutedSlice(getOutputMap(), offsets, sizes);
-
-  Value scale = getScale();
-
   SmallVector<Value> tiledOperands;
+
+  SmallVector<Range> querySlice = getPermutedSlice(getQueryMap(), offsets, sizes);
   tiledOperands.emplace_back(getSlice(builder, loc, getQuery(), querySlice));
-  tiledOperands.emplace_back(getSlice(builder, loc, getKey(), keySlice));
+
+  SmallVector<Range> keySlice = getPermutedSlice(getKeyMap(), offsets, sizes);
+   tiledOperands.emplace_back(getSlice(builder, loc, getKey(), keySlice));
+
+  SmallVector<Range> valueSlice = getPermutedSlice(getValueMap(), offsets, sizes);
   tiledOperands.emplace_back(getSlice(builder, loc, getValue(), valueSlice));
+
+  Value scale = getScale();
   tiledOperands.emplace_back(scale);
+
+  std::optional<Value> attnMask = getMask();
+  if (attnMask) {
+    SmallVector<Range> maskSlice = getPermutedSlice(*getMaskMap(), offsets, sizes);
+    tiledOperands.emplace_back(getSlice(builder, loc, *attnMask, maskSlice));    
+  } else {
+    tiledOperands.emplace_back(Value());
+  }
```

Comment:
We shouldn't have this because we are already handling this in the offsets when doing `tiledOperands[maskSlice ? 5 : 4].`

Reply:
Addressed.